dev/io.openliberty.java.internal_fat/src-reference/java23/io/openliberty/java/internal/package-info.java

```diff
@@ -0,0 +1,13 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ * IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.java.internal;
```

Comment:
In the misc repo you also have a `module-info.java` file.  Perhaps that should also be included here.

Reply:
Good catch!  I have added that file here and for the others as well.